libjwt/jwt-gnutls.c

```diff
@@ -0,0 +1,223 @@
+/* Copyright (C) 2017 Nicolas Mora <mail@babelouest.org>
+	 This file is part of the JWT C Library
```

Comment:
I think you were too quick with the find/replace here. The copyright doesn't need tab indents. Just copy what's in `jwt.c` please.